docs/custom-filters-v1.md

```diff
@@ -0,0 +1,47 @@
+# Custom filters V1
+
+Custom filters are Javascript code snipets that will be embedded into gitStream's functionality after a review and approval proccess by our engineers
+
+!!! attention
+
+    :octicons-beaker-24: Coming soon
+
+
+###  Creating filters
+
+Filters can have at most one input parameter such as the available [context variables](context-variables.md), that is provided to the filter in the `.cm` file using a pipe.
+
```

Comment:
Filters can have as many inputs arguemnts you wish. The first input argument can be "piped", when you have more than one you can use the more conventinoal function arguments, e.g. `f(input1, input2)` shall be used like `{{ input1 | f(input2) }}`